src/cljs_rest/core.cljs

```diff
@@ -118,15 +119,50 @@
     m
     m))
 
+(def link-pattern
+  "Matches a `link-value` according to https://tools.ietf.org/html/rfc5988#section-5,
+  capturing `URI-Reference` and a `rel` value for `link-param`, ignoring all other
+  `link-param` fields if present."
+  #"(?i)<(.*?)>(?:;\s*(?!rel)[^=]+=\".*?\")*;\s*rel=\"(.*?)\"(?:;\s*[^=]+=\".*?\")*")
```

Comment:
But this is basically unmaintainable so I plan to try to break this into stages, I just don't want a super naive pattern like splitting on comma, which could break for all sorts of reasons.